examples/tpu_training-tf.ipynb

```diff
@@ -0,0 +1,904 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #12.      tf_dataset = tf_dataset.shuffle(len(dataset)).batch(BATCH_SIZE, drop_remainder=True)
</pre><p>Can spend one sentence on why <code>drop_remainder</code> is set to True.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/notebooks/pull/313/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='45'/>

Reply:
<p>I added this in the first place where drop_remainder is used!</p>